src/stuff.m4

```diff
@@ -731,12 +735,14 @@ m4_define([_MAKE_OPTARG_SIMPLE_CASE_SECTION], [m4_do(
 		[bool],
 		[_JOIN_INDENTED(_INDENT_LEVEL_IN_ARGV_CASE_BODY,
 			[[$5="on"]],
+			[_supplied[$5]=1],
```

Comment:
```
- [_supplied[$5]=1],
+ [[_supplied$5=1]],
```

For the sake of consistent style, please put the whole assignment (and other occurrences as well)  into double-qoutes - the only reason why to single-quote is when the single-quoted part is a macro, which this is not.